src/IECoreImage/DisplayDriverServer.cpp

```diff
@@ -423,9 +437,39 @@ void DisplayDriverServer::Session::handleReadOpenParameters( const boost::system
 
 		const StringData *displayType = parameters->member<StringData>( "remoteDisplayType", true /* throw if missing */ );
 
+		const BoolData *mergeDriverData = parameters->member<BoolData>( "mergeDriver", false);
```

Comment:
I would suggest omitting the `mergeDriver` parameter completely as it doesn't seem to add anything. We could just use the existence of `sessionId` to enable the merging behaviour.